cleanlab_studio/utils/tlm_lite.py

```diff
@@ -206,12 +218,6 @@ def _batch_score(
         if all(isinstance(score, dict) for score in score_response):
             score_response = cast(List[TLMScore], score_response)
             return [{"response": res, **score} for res, score in zip(response, score_response)]
-        elif all(isinstance(score, (float, int)) for score in score_response):
-            score_response = cast(List[float], score_response)
-            return [
-                {"response": res, "trustworthiness_score": score}
-                for res, score in zip(response, score_response)
-            ]
         else:
             raise ValueError(f"score_response has invalid type")
```

Comment:
Same here re needing the else statement

Reply:
this is also just trying to pass the mypy check